pkg/apis/registries/v1alpha1/instance_types.go

```diff
@@ -17,6 +16,15 @@ const (
 	InstanceStatusPhaseError       InstanceStatusPhaseName = "Error"
 )
 
+const (
+	ScheduleTypeHourly   string = "Hourly"
+	ScheduleTypeDaily    string = "Daily"
+	ScheduleTypeWeekly   string = "Weekly"
+	ScheduleTypeCustom   string = "Custom"
+	ScheduleTypeManually string = "Manually"
+	ScheduleTypeNone     string = "None"
+)
```

Comment:
Maybe define a `ScheduleType` type for this?

```suggestion
type ScheduleType string

const (
	ScheduleTypeHourly   ScheduleType = "Hourly"
	ScheduleTypeDaily    ScheduleType = "Daily"
	ScheduleTypeWeekly   ScheduleType = "Weekly"
	ScheduleTypeCustom   ScheduleType = "Custom"
	ScheduleTypeManually ScheduleType = "Manually"
	ScheduleTypeNone     ScheduleType = "None"
)
```